src/components/App.js

```diff
@@ -7,7 +7,7 @@ import '../css/App.css';
 
 const stats = require('../stats.json');
 const years = Object.keys(stats).map(k => k.substring(0, 4)).reduce((p, c) => {
-  if (!p.includes(c)) p.push(c);
+  if (p.indexOf(c) === -1) p.push(c);
```

Comment:
I think not but I can confirm that this works (on IE11) with this change.

I'm a bit confused because there is obviously some Babel-like translation being done or I'd have far worse browser support than I do but for some reason `includes` and `padLeft` are not covered.

I guess I'd have to look in to `react-scripts` to see why.